CompilerSource/filesystem/file_find.cpp

```diff
@@ -1,158 +0,0 @@
-/********************************************************************************\
```

Comment:
This source file was something else? It's not related to `std::filesystem` like the name implied. I'm not sure if @JoshDreamland is ok with deleting this?

Reply:
It was just used to itreate directories which we do with a directory iterator now